plugins/org.eclipse.emf.mwe.core/src/org/eclipse/emf/mwe/internal/core/ast/util/converter/StringArrayConverter.java

```diff
@@ -12,7 +12,8 @@
 
 public class StringArrayConverter implements Converter<String[]> {
 
-    public String[] convert(final String s) {
+    @Override
```

Comment:
Maybe just add a separate commit that reformats the entire repo? Is the code style shared?